daffodil-lib/src/main/scala/org/apache/daffodil/xml/DaffodilConstructingLoader.scala

```diff
@@ -72,7 +72,7 @@ object Position {
  * gets positions is different. It is given just an offset into the document file/stream,
  * and it therefore must synthesize line number/col number info itself.
  */
-class DaffodilConstructingLoader(uri: URI, errorHandler: org.xml.sax.ErrorHandler, addLineColInfo: Boolean = false)
+class DaffodilConstructingLoader(uri: URI, errorHandler: org.xml.sax.ErrorHandler, addLineColInfo: Boolean = true)
```

Comment:
Is it worth removeing the addLineColInfo paramter? We never use it anywhere.

Reply:
Will remove